README.md

```diff
@@ -6,3 +6,36 @@
 
 Helper website for the text-based online RPG Torn City https://www.torn.com/
 The website is hosted here: https://yata.alwaysdata.net/
+
+# Local Setup Instructions
+
+
+
+## Setup
+    git clone https://github.com/Kivou-2000607/yata.git
+
+    cd yata
+
+    pip install -r requirements.txt
+
+Create a local .env file
+
+    SECRET_KEY="SUPER_SECRET_KEY"
+    CACHE_RESPONSE=10
+    APIKEY="TORNAPIKEY"
+    
+Then run setup.py to initalise everything
+
+    python setup.py
+
+
+
+## Running YATA
+
+To emulate cron activity _dev_cron.py_ can be run as a seperate process. Cron jobs designed to run on a per minute basis will be run as such. Cron's with a longer delay will run on a 30 minute schedule.
+
+    python dev_cron.py
+
+To launch the application simple start the Django Application
+
+    python manage.py runserver
```

Comment:
We should add that:

- administration login/psw are admin/adminpass 
-  for item Default item markets should be added from the administration section with the "Put on market" action

Reply:
Can you expand a little bit for me on that second point? It's not something I've noticed

Reply:
Updated